services/debugger/destination/eventDeliveryStatusUploader_test.go

```diff
@@ -212,6 +216,9 @@ var _ = Describe("eventDeliveryStatusUploader", func() {
 
 	Context("RecordEventDeliveryStatus", func() {
 		It("returns false if disableEventDeliveryStatusUploads is false", func() {
+			wg := sync.WaitGroup{}
+			wg.Add(1)
```

Comment:
A WaitGroup seems a bit overkilling for this, it looks like it could be done with a simple channel and a `<-time.After()`. No big deal though given that it's a test.